content/_data/changelogs/weekly.yml

```
@@ -17985,7 +17960,17 @@
         pr_title: Correctly encode pound sign in `sortable.js`
         message: |-
           Fix sorting of British currency in tables.
-
+      - type: rfe
+        category: developer
+        pull: 7051
+        authors:
+          - janfaracik
+          - timja
+        pr_title: Bring native support for side panel app bars, sticky side bar and
+          changing app bar heading level
+        message: |-
+          - Developer: Add support for the app-bar component in the sidepanel.
```

The dash at the beginning is inconsistent with the other messages. Also, "sidepanel" is not an English word.

```suggestion
          Developer: Add support for the app-bar component in the side panel.
```